tethne/classes/streaming.py

```diff
@@ -102,6 +111,9 @@ def __iter__(self):
         return PList(self)
 
     def __init__(self, *args, **kwargs):
+        """
+        Refer :meth:`.Corpus.__init__` for argument description and usage.
```

Comment:
Sphinx ignores the constructor when auto-generating API documentation, so this kind of thing should go in the class docstring.